CHANGELOG.md

```diff
@@ -0,0 +1,2 @@
+Please see [doc/docs/project/CHANGELOG.md
```

Comment:
But why doesn't that work if we just put all of the markdown files directly in `doc/`?